src/utils/newrelic.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+let _newrelic;
+
+// NewRelic initialization
+function init() {
+  if (process.env.NEW_RELIC_ENABLED) {
+    process.env.NEW_RELIC_NO_CONFIG_FILE = 'True';
+
+    if (!_newrelic) {
+      // newrelic is cached as a singleton.
+      _newrelic = require('newrelic');
+    }
+  }  
+  return _newrelic;
+}
+
+function crashClose() {
+  if (!_newrelic) {
+    return Promise.resolve();
+  } else {
```

Comment:
this is just a style issue but IMHO if you ``return`` in the ``if`` it's better not to use ``else`` - just reduces some nesting and less curly braces.